lib/maxmind/geoip2/version.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module MaxMind
+  module GeoIP2
+    # The Gem version.
+    VERSION = '1.1.0'
```

Comment:
Would it be possible to use this in the gemspec? See [here](https://github.com/maxmind/minfraud-api-ruby/blob/4ef0dc381c9f73a516e0a4a4f7f21a00874e9d22/minfraud.gemspec#L10) for what we do in the `minfraud` gem. The hope would be the version is in fewer spots in the repo.